moped-database/views/project_geography.sql

```diff
@@ -23,6 +23,7 @@ FROM moped_project
 INNER JOIN moped_proj_components ON moped_project.project_id = moped_proj_components.project_id
 INNER JOIN moped_components ON moped_proj_components.component_id = moped_components.component_id
 INNER JOIN feature_layers ON moped_components.feature_layer_id = feature_layers.id
-INNER JOIN uniform_features ON moped_proj_components.project_component_id = uniform_features.component_id;
+INNER JOIN uniform_features ON moped_proj_components.project_component_id = uniform_features.component_id
+WHERE moped_proj_components.is_deleted IS false;
 
-COMMENT ON VIEW public.project_geography IS 'The project_geography view merges project-specific data with the unified geographical features from the uniform_features view. It links projects with their respective geographical components, including type, attributes, and location.';
+COMMENT ON VIEW public.project_geography IS 'The project_geography view merges project-specific data with the unified geographical features from the uniform_features view. It links projects with their respective geographical components, including type, attributes, and location.'; -- noqa: LT05
```

Comment:
This is what it looks like to carve out an exception to a linting rule. This line is /way/ too long, but I think it's the lesser of two evils than doing some convoluted use of `' ||<newline>'` or something like that to break the long string up. 

I do want to keep this linting rule, I think it's always better to fit in an 80 character terminal if you can, and this rule will keep us breaking our long clauses up over lines, which is much clearer to me.

Reply:
agreed!